src/libmeasurement_kit/ooni/utils.cpp

```diff
@@ -116,6 +147,9 @@ ErrorOr<json> geoip(std::string ip, std::string path_country,
     node["country_code"] = country_code.as_value();
     node["country_name"] = country_name.as_value();
     node["asn"] = asn.as_value();
+    if (path_city != "") {
+        node["city_name"] = city_name.as_value();
+    }
```

Comment:
I am not sure it makes sense to include the city name only if a path for the city was specified, however we all agreed that this API returning a JSON sucks and should be removed, so I'll let it pass.